mozilla/fx-private-relay
# for MPP-3817: prevent all Relay operations when user.is_active = False

This PR fixes #MPP-3817.

How to test:
1. Run new `python manage.py deactivate_user` a number of ways (See `--help` for details)
2. After deactivating a user, make sure they cannot perform any of the following:
   * [ ] Existing sessions are signed out
   * [ ] Cannot sign back in
   * [ ] Cannot make any API requests
   * [ ] Cannot receive any emails
   * [ ] Cannot reply to any previous emails
   * [ ] Cannot receive any text messages
   * [ ] Cannot receive any forwarded phone calls

Note: some tests may be easiest on the dev server.
